04Lesson/01Homework/index.html

```diff
@@ -2,35 +2,40 @@
 <html>
 
 <head>
-    <title></title>
+    <title>Lesson 4 Homework Profession Portal</title>
     <link rel="stylesheet" href="css/style.css">
 </head>
 
 <body>
     <header>
         <h1>Profession Portal</h1>
     </header>
-    <div id="navigation">
+    <div id="navigation" class="clearfix">
         <nav>
             <a class="nav-link" href="Home">Home</a>
             <a class="nav-link" href="Companies">Companies</a>
             <a class="nav-link" href="Positions">Positions</a>
         </nav>
+        </header>
+        <main id="content">
```

Comment:
You open your <main> tag inside of the div you use for the navigation and then close the div for the navigation after you open the main. This is causing your layout to not render the way you want it. I suggest fixing this.